model/components/micrometer.yml

```diff
@@ -0,0 +1,19 @@
+---
+components:
+
+  # micrometer
+  - id: micrometer
+    name: Micrometer
+    description: Micrometer provides a simple facade over the instrumentation clients for the most popular monitoring systems, allowing you to instrument your JVM-based application code without vendor lock-in. Think SLF4J, but for metrics.
+    link: https://micrometer.io/
+    logo: micrometer
+    categories:
+      - agent
```

Comment:
I'm just thinking: maybe we should introduce a new category for "instrumentation libraries". Same would fit for Prometheus, Jaeger, Zipkin, etc. Because all of them are not really agents. @ivansenic: what do you think?

Reply:
Yea it's fine, let's do it like this. Makes sense imo.

Reply:
https://github.com/openapm/landscape-model/issues/24